src/containers/chat/chat.tsx

```diff
@@ -94,16 +92,25 @@ export const Chat: FC<Props> = ({ roomId, room }) => {
   const isChatDisabled = stage !== "chat";
```

Comment:
Better to think in positive terms: `isChatEnabled` otherwise is inverse logic when you're checking for the double negative `!isChatDisabled` actually means that is enabled.... harder to read.